templates/documentation.html

```diff
@@ -0,0 +1,297 @@
+                  <i class="fa fa-fw fa-tags"></i>
+                  Releases
+                </a>
+                <a class="list-group-item" href="https://invenio.readthedocs.io/en/latest/upgrading/index.html">
+                  <i class="fa fa-fw fa-angle-double-up"></i>
+                  Upgrading
+                </a>
+              </div>
+            </div>
+            <div class="col-sm-4">
+              <h3 class="text-center">
+                <a href="https://inveniordm.docs.cern.ch/">
+                  <img class="documentation-logo" src="{{ '/static/img/logo-invenio-rdm.svg'|asseturl }}"
+                       title="InvenioRDM" alt="InvenioRDM"/>
+                </a>
+              </h3>
+              <div class="panel panel-default">
+                <div class="panel-heading">InvenioRDM</div>
+                <div class="list-group">
+                  <a class="list-group-item" href="https://inveniordm.docs.cern.ch/develop/">
+                    <i class="fa fa-fw fa-play-circle"></i>
+                    Getting started
+                  </a>
+                  <a class="list-group-item" href="https://inveniordm.docs.cern.ch/">
+                    <i class="fa fa-fw fa-book"></i>
+                    Documentation
+                  </a>
+                </div>
+              </div>
+              <div class="list-group">
+                <a class="list-group-item" href="https://inveniordm.docs.cern.ch/deployment/">
+                  <i class="fa fa-fw fa-rocket"></i>
+                  Deploy
+                </a>
+                <a class="list-group-item" href="https://inveniordm.docs.cern.ch/extensions/">
+                  <i class="fa fa-fw fa-cubes"></i>
+                  Extend
+                </a>
+              </div>
+            </div>
+            <div class="col-sm-4">
+              <h3 class="text-center">
+                <a href="https://invenioils.docs.cern.ch/">
+                  <img class="documentation-logo" src="{{ '/static/img/logo-invenio-ils.svg'|asseturl }}"
+                       title="InvenioILS" alt="InvenioILS"/>
+                </a>
+              </h3>
+              <div class="panel panel-default">
+                <div class="panel-heading">InvenioILS</div>
+                <div class="list-group">
+                  <a class="list-group-item" href="https://invenioils.docs.cern.ch/quickstart/">
+                    <i class="fa fa-fw fa-play-circle"></i>
+                    Getting started
+                  </a>
+                  <a class="list-group-item" href="https://invenioils.docs.cern.ch/">
+                    <i class="fa fa-fw fa-book"></i>
+                    Documentation
+                  </a>
+                </div>
+              </div>
+              <div class="list-group">
+                <a class="list-group-item" href="#">
+                  <i class="fa fa-fw fa-rocket"></i>
+                  Deploy
```

Comment:
cc @ntarocco do you guys something to link to here and in `Advanced customizations`? Otherwise, we can remove these for now as I think the rest of the page is ready to go online.

Reply:
We don't have something right now, our documentation is still a WIP. I also agree to remove it then.